content/docs/instrumenting/exporters.md

```diff
@@ -92,6 +92,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [NSQ exporter](https://github.com/lovoo/nsq_exporter)
    * [Mirth Connect exporter](https://github.com/vynca/mirth_exporter)
    * [MQTT blackbox exporter](https://github.com/inovex/mqtt_blackbox_exporter)
+   * [MQTT push gateway](https://github.com/hikhvar/mqtt2prometheus)
```

Comment:
The text here should just be the exporter name

Reply:
Fixed it.